app/controllers/member/messages_controller.rb

```diff
@@ -2,29 +2,13 @@ class Member::MessagesController < ApplicationController
   before_action :authenticate_member!
 
   def create 
-    @message = Message.new(message_params)
-    @message.member_id = current_member.id 
+    @message = Message.message_create!(current_member, message_params)
```

Comment:
`message_create!` メソッドですが、普通にRailsの機能を使って、下記のように実装する方がシンプルではないでしょうか？

```suggestion
    @message = current_member.messages.create!(message_params)
```